cartridges/int_stickyio_sfra/cartridge/templates/default/checkout/customer/customerCard.isml

```diff
@@ -0,0 +1,72 @@
+	                <button class="btn btn-primary btn-block submit-customer" type="submit" name="submit" value="submit-customer">
+	                    ${Resource.msg('button.guest.customer.continue', 'checkout', null)}
+	                </button>
+	            </isif>
+	             <strong>${Resource.msg('msg.guest.customer.exists', 'checkout', null)} <button type="button" class="btn btn-link js-login-customer">${Resource.msg('action.customer.login', 'checkout', null)}</button></strong>
+	        </fieldset>
+	        <input type="hidden" name="${pdict.csrf.tokenName}" value="${pdict.csrf.token}"/>
+	    </form>
+    </isif>
+    <form autocomplete="on" id="registered-customer" method="POST" class="registered-form <isif condition="${pdict.allowGuestCheckout}"> d-none</isif>" action="${URLUtils.url('CheckoutServices-LoginCustomer')}"
+        <isprint value=${pdict.forms.registeredCustomerForm.attributes} encoding="off" /> novalidate>
+        <fieldset class="customer-information-block">
+            <div class="row">
+                <div class="col-sm-12">
+                    <div class="form-group">
+                        <label class="form-control-label">
+                            ${Resource.msg('field.customer.email','checkout',null)}
+                        </label>
+                        <input type="text" class="form-control email" id="email"
+                                    value="${(pdict.customer.registeredUser && pdict.customer.profile && pdict.customer.profile.email) ? pdict.customer.profile.email : ''}" <isprint value=${pdict.forms.registeredCustomerForm.email.attributes} encoding="off"/>
+                                    aria-describedby="emailInvalidMessage" >
+                        <div class="invalid-feedback" id="emailInvalidMessage"></div>
+                    </div>
+                    <div class="form-group password-form">
+                        <label class="form-control-label">
+                            ${Resource.msg('field.customer.password','checkout',null)}
+                        </label>
+                        <input type="password" class="form-control password" id="password"
+                                    <isprint value=${pdict.forms.registeredCustomerForm.password.attributes} encoding="off"/>
+                                    aria-describedby="passwordInvalidMessage" >
+                        <div class="invalid-feedback" id="passwordInvalidMessage"></div>
+                    </div>
+                </div>
+            </div>
+            <div class="row">
+                    <div class="col-12">
+                        <div class="mb-sm-3">
+                            <button class="btn btn-primary btn-block submit-customer-login" type="submit" name="submit" value="submit-login">
+                                ${Resource.msg('button.customer.login', 'checkout', null)}
+                            </button>
+							<isif condition="${pdict.allowGuestCheckout}">
```

Comment:
Please correct spacing issues in line 60?

Reply:
fixed